.bazelrc

```diff
@@ -40,9 +40,8 @@ build:release --workspace_status_command="node ./tools/bazel-stamp-vars.js"
 # Temporary Settings for Ivy   #
 ################################
 
-# Use the legacy AOT compiler strategy. We don't want to compile with Ivy nor with "ngtsc" which
-# does not generate factory files which are needed for AOT.
-build --define=compile=legacy
+# Use the Ivy AOT compiler strategy. We want to compile with Ivy and with "ngtsc".
+build --define=compile=aot --keep_going=true
```

Comment:
Why add `--keep_going=true` here? This should _not_ be part of the default bazelrc. Especially not of this section which is primarily about the compile mode for `ng_module.bzl`

Reply:
It's just for debugging and fixing in this branch. I wasn't planning to leave it. Is there a location that it would make sense to put a toggle for it or a better place to leave a comment about where it should be enabled when needed?